backend/app/GraphQL/Mutations/CreateAnswer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+
+use App\Models\Answer;
+use App\Models\Question;
+use App\Models\User;
+use Exception;
+use Illuminate\Support\Facades\Validator;
+
+final class CreateAnswer
+{
+    /**
+     * @param  null  $_
+     * @param  array{}  $args
+     */
+    public function __invoke($_, array $args)
+    {
+        try {
+            $check_user = User::findOrFail($args['user_id']);
+            $check_question = Question::findOrFail($args['question_id']);
+
+            $validator = Validator::make($args, [
+                'content' => ['required'],
+                'user_id' => ['required'],
+                'question_id' => ['required']
+            ]);
+            if ($validator->fails()) {
+                return $validator->messages();
+            }
+            $newAnswer = Answer::create([
+                'content' => $args['content'],
+                'user_id' => $check_user->id,
+                'question_id' => $check_question->id,
+            ]);
+            return $newAnswer;
```

Comment:
```suggestion
          $validator = Validator::make($args, [
                'content' => ['required'],
                'user_id' => ['required'],
                'question_id' => ['required']
            ]); // creating validation
            
            if ($validator->fails()) {
                return $validator->messages();
            } // checking if validation failed
            
            $newAnswer = Answer::create([
                'content' => $args['content'],
                'user_id' => $check_user->id,
                'question_id' => $check_question->id,
            ]); // creating new user
            
            return $newAnswer; // return user
```

You can add spaces between blocks of code if they have different purposes so that it will be easier to read